docs/user/reference/openapi.json

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
I've made an issue to deal with the fact that this in a different markup langauge to the AyncAPI spec. Not merge-blocking, can be dealt with later: https://github.com/DiamondLightSource/blueapi/issues/181